Task/Taskfile.yml

```diff
@@ -57,6 +57,30 @@ tasks:
                    --env INPUT_LOG_LEVEL="{{.INPUT_LOG_LEVEL}}" \
                    --rm -v "{{.ROOT_DIR}}:/goat" seiso/goat:latest
 
+  validate:
+    desc: Validate pre-commit config and hooks files
+    cmds:
+      - task: validate-pre-commit-config
+      - task: validate-pre-commit-manifest
+
+  validate-pre-commit-config:
+    desc: Validate the pre-commit configuration file
+    dir: ../../..
+    cmds:
+      - echo "-Validating pre-commit config file"
+      - pipenv run pre-commit validate-config '.pre-commit-config.yaml'
+    status:
+      - test -f ../../.pre-commit-config.yaml
```

Comment:
Why are these two different?  Also, was something wrong with the `find` approach, just using `../..` instead?  I didn't test it, so wasn't yet sure if there were any issues, just wondering

Reply:
Different, as in the quotes.  Since it's hard coded we can remove them from the other